service/src/main/java/tennisclub/service/CourtServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package tennisclub.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import tennisclub.dao.CourtDao;
+import tennisclub.entity.Court;
+import java.util.List;
+
+@Service
+public class CourtServiceImpl implements CourtService {
+    private final CourtDao courtDao;
+
+    @Autowired
+    public CourtServiceImpl(CourtDao courtDao) {
```

Comment:
Is the constructor needed though? Doesn't it work fine without it? 🤔 in the reference implementation there isn't a constructor and `@Inject` annotation is used instead 🤔